tools/firtool/firtool.cpp

```diff
@@ -118,6 +118,11 @@ static cl::opt<bool>
                           "Grand Central annotations"),
                  cl::init(false));
 
+static cl::opt<bool>
+    checkCombCycles("firrtl-check-comb-cycles",
+                    cl::desc("check combinational cycles on firrtl"),
+                    cl::init(false));
```

Comment:
For this to catch some cases that the SFC will also catch, this will need to run before IMCP (since IMCP will aggressively prune cycles). Does this continue to work if you move it up there?

Reply:
Moved this pass up before IMCP. It works well!